tools/build_release.py

```diff
@@ -179,18 +223,28 @@
                     build_profile=extract_profile(parser, options, toolchain),
                 )
 
-
     # copy targets.json file as part of the release
-    copy(join(dirname(abspath(__file__)), '..', 'targets', 'targets.json'), MBED_LIBRARIES)
+    copy(
+        join(dirname(abspath(__file__)), '..', 'targets', 'targets.json'),
+        MBED_LIBRARIES
+    )
 
     # Write summary of the builds
     if options.report_build_file_name:
-        file_report_exporter = ReportExporter(ResultExporterType.JUNIT, package="build")
-        file_report_exporter.report_to_file(build_report, options.report_build_file_name, test_suite_properties=build_properties)
-
-    print "\n\nCompleted in: (%.2f)s" % (time() - start)
-
-    print_report_exporter = ReportExporter(ResultExporterType.PRINT, package="build")
+        file_report_exporter = ReportExporter(
+            ResultExporterType.JUNIT, package="build"
+        )
+        file_report_exporter.report_to_file(
+            build_report,
+            options.report_build_file_name,
+            test_suite_properties=build_properties
+        )
+
+    print("\n\nCompleted in: (%.2f)s" % (time() - start))
```

Comment:
Oh gosh, I missed one.

```suggestion
    print("\n\nCompleted in: ({0:.2f})s".format(time() - start))
```

References (because I didn't know how this worked until now):
- https://stackoverflow.com/questions/455612/limiting-floats-to-two-decimal-points
- https://docs.python.org/3/library/string.html#format-specification-mini-language